cluster.yml

```diff
@@ -30,15 +30,24 @@
     - { role: docker, tags: docker }
     - { role: rkt, tags: rkt, when: "'rkt' in [ etcd_deployment_type, kubelet_deployment_type ]" }
 
-- hosts: etcd:!k8s-cluster
+- hosts: all
```

Comment:
**All** nodes in inventory might be covering nodes that are out of scope of Kargo, please use ``k8s-cluster:etcd:calico-rr`` instead.